db/migrate/20150605192639_add_postgrest_auth_table.rb

```diff
@@ -0,0 +1,36 @@
+class AddPostgrestAuthTable < ActiveRecord::Migration
+  def up
+    execute <<-SQL
+      CREATE SCHEMA postgrest;
```

Comment:
we i'll use the schema to handle with api business logic and handle with api access on schema 1?